_episodes/04-data-types-and-format.md

```diff
@@ -15,7 +15,11 @@ objectives:
     - "Analyze datasets having missing/null values (NaN values)."
     - "Write manipulated data to a file."
 keypoints:
-    - "FIXME"
+    - "Pandas and base Python use slightly different names for data types"
+    - "Each column in a DataFrame has a shared data type for all rows"
+    - "The type of one column in a DataFrame can be checked using `dataFrameName[column_name].dtype`"
```

Comment:
`The type of data...`

Reply:
I also have a preference for keeping code out of the key points, and this seems to be shared across other lessons. Could we do:

"The data type of one column in a DataFrame can be checked using the column's `dtype` attribute"?

Reply:
Columns do not have type. Data in columns does. I suggest something like`The type of data in a single column in a DataFrame...`

Reply:
Oh, I thought dtype was an attribute of the column object. Does `.dtype` check for the most inclusive type?